src/virtual_dom/node/el.rs

```diff
@@ -205,15 +205,15 @@ impl<Ms> El<Ms> {
 
 /// Allow the user to clone their Els. Note that there's no easy way to clone the
 /// closures within listeners or lifestyle hooks, so we omit them.
-impl<Ms> Clone for El<Ms> {
+impl<Ms: Clone> Clone for El<Ms> {
```

Comment:
I hoped we can remove all `Clone` restrictions for `Ms` because it force us to do some ugly things like wrapping `Msg` variants to `Rc` in some applications.

Reply:
Same stroy for `Node<Ms>`